weave-js/src/components/Panel2/PanelPlot/versions/index.ts

```diff
@@ -56,5 +58,5 @@ export type ContinuousSelection = v11.ContinuousSelection;
 export type DiscreteSelection = v11.DiscreteSelection;
 export type AxisSelections = v11.AxisSelections;
 
-export type ConcretePlotConfig = v14.ConcretePlotConfig;
+export type ConcretePlotConfig = v16.ConcretePlotConfig;
```

Comment:
i wish this was something that was read off of the `migrate` return value and not need to be manually bumped